Technical-Steering-Committee/proposal.md

```diff
@@ -0,0 +1,89 @@
+
+The initial voting members of the TSC will consist of representatives from the Zowe Leadership Committee (ZLC) as well as the current lead or representative from each squad, as defined below. Where the squad lead is also one of the ZLC representative this position should be filled by another squad SME.
+
+  - Joe Winchester (ZLC representative)
+  - Sean Grady (ZLC representative)
+  - Mark Ackert (ZLC representative)
+  - _TBD_ - (Zowe App Framework/Desktop squad)
+  - Mike Bauer (Zowe CLI Squad)
+  - Fernando Rijo Cedeno (Zowe Explorer squad)
+  - Elliot Jalley (Zowe API Squad)
+  - _TBD_ (Zowe Systems - CI/CD squad)
+  - Nan Nan Li (Zowe Documentation squad)
+  - _TBD_ (Zowe Onboarding squad)
+
+The TSC voting members will have no more than one-third (33%) of voting members affiliated with the same employer. In the case where a TSC voting member has an employment change which violates this requirement, the TSC voting member should be replaced by the body in which thier status is provided.
+
+The TSC voting members may adjust the makeup of the TSC voting members after the initial year with approval of the current Zowe Committers. It is expected that any makeup changes would look to align with the growing technical community structure to ensure equitable representation. Any TSC voting member elections will be facilated by the Open Mainframe Project staff to ensure vendor-neutrality. 
+
+TSC voting members are expected to regularly participate in TSC meetings and other activities. TSC voting members may voluntarily resign at any time, in which case the squad through which the voting member was elected will be responsible for providing a new voting member. In the exceptional situation where the TSC determines a voting member is acting in a manner detrimental to the Open Mainframe Project or Zowe project, the other TSC voting members will consult with Open Mainframe Project staff to determine a course of action.
+
+## Operations and Processes
+
+It is expected that the TSC will establish it's general operations and policies, clearly defined in the [GOVERNANCE.md](../GOVERNANCE.md), but in general the TSC will do the following.
+
+### Chairperson and Secretary
+
+The TSC will elect a chairperson amongst the TSC voting members which will have the responsibilities and duties of...
+
+- Organizing and conducting meetings, and coordinating responses to any voting issues.  
+- Be the primary representative of the Zowe project to the Open Mainframe Project Techincal Advisory Counsel (TAC).
+- Be the primary spokesperson for the Zowe project.
+
+The TSC chairperson may choose from time-to-time to delegate these duties to other TSC voting members if they are unable to fulfill these duties, or if they deem another SME is more appropriate.
```

Comment:
This is a shift of the TSC chairperson to be the more prominent representative of the Zowe project, which aligns with how other OMP and LF projects work.

Reply:
I agree - it's good to have a central spokesperson for technical issues covering TAC.  For the OMP itself though, the ZLC has a chairperson (currently Bruce).  I see other charters where there is a project CEO who is the representative - would it make sense to separate TSC chair being technical rep to the TAC and ZLC chair being business rep ?

Reply:
I've not seen projects we host have a "CEO" role within the technical leadership structure. If you see examples feel free to point me to them.

Generally in projects we leave the decision to separate the spokesperson and/or TAC representative roles to different individuals to the TSC and/or chair to decide. It usually comes down to a bandwidth or comfortability question for the chair itself, and most often in projects we see the chair look to spread opportunities to be a voice of the community to other TSC members.